generate.py

```diff
@@ -142,13 +142,21 @@ def generate_image(text, fp):
 
 
 if __name__ == "__main__":
+    import argparse
     import sys
     import tempfile
 
-    fd, filename = tempfile.mkstemp(".png")
-    text = " ".join(sys.argv[1:])
+    parser = argparse.ArgumentParser(description='Generate an Is This A Pigeon? image.', epilog='Example: ./generate.py me: Is \\"a single example\\" good documentation?')
```

Comment:
Not sure about the `description` and `epilog`. Could we leave this blank for now?